garrysmod/gamemodes/terrortown/gamemode/cl_hudpickup.lua

```diff
@@ -27,7 +27,7 @@ function GM:HUDWeaponPickedUp( wep )
    pickup.fadein    = 0.04
    pickup.fadeout   = 0.3
 
-   local role = client.GetRole and client:GetRole() or ROLE_INNOCENT
+   local role = LocalPlayer().GetRole and LocalPlayer():GetRole() or ROLE_INNOCENT

```

Comment:
Yes, because there's a chance that GetRole doesn't exist on the player, for some fucking reason